src/Spago/Config.purs

```diff
@@ -464,6 +465,14 @@ shouldComputeNewLockfile { workspace, workspacePackages } workspaceLock =
           _ -> false
       )
 
+getGlobalGitCacheForRepo :: Core.GitPackage -> FilePath
```

Comment:
This function should be moved in the `where` block of `getGitPackageInLocalCache`, since it's only used there